components/RedesignedPricing/PricingCalculator.tsx

```diff
@@ -173,17 +195,17 @@ function PlanSummary({ plan, price }: { plan: Plan; price: string }) {
 
 export function PricingCalculatorPage() {
   const [runs, setRuns] = useState(50000);
-  const [steps, setSteps] = useState(20);
+  const [steps, setSteps] = useState(3);
```

Comment:
Why is this 3? Should we just make this 50000?

Reply:
@jakobevangelista, following up on this 👆🏼